server_addon/blender/client/ayon_blender/plugins/publish/extract_camera_abc.py

```diff
@@ -53,6 +67,12 @@ def process(self, instance):
                 flatten=True
             )
 
+        scene.frame_start = frame_start
+        scene.frame_end = frame_end
+        scene.frame_step = frame_step
+        scene.render.fps = fps
+        scene.render.fps_base = fps_base
```

Comment:
Maybe it is a good idea to implement the function in the library to set all these frame_range related settings?